### PR TITLE
Fixing Issues with Code

### PR DESCRIPTION
**Pull request by tylerhyperHD**
I exposed ur code's problems lol

https://youtu.be/zx29OjreriU

### CODE REVIEW

**Comment by IvanTheOven** [comment]
first of all, was it really necessary to make a video? you're literally just exposing a ban.java file and a pom file.

**Comment by OxLemonxO** [comment]
So you say we don't know how to code? I'm a developer of the official TotalFreedomMod lol.

File: src/main/java/me/unraveledmc/unraveledmcmod/command/Command_linkdiscord.java [hunk 1]
```diff
@@ -37,7 +37,10 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         {
             String code = "";
             Random random = new Random();
-            for (int i = 0; i < 5; i++) code += random.nextInt(10);
+            for (int i = 0; i < 5; i++)
+            {
+                code += random.nextInt(10);
```
Comment (OxLemonxO) [inline src/main/java/me/unraveledmc/unraveledmcmod/command/Command_linkdiscord.java]: why are you removing this? lambda expressions are fine.,

Comment (tylerhyperHD) [inline src/main/java/me/unraveledmc/unraveledmcmod/command/Command_linkdiscord.java]: That was a result of me pressing Source > Format, which then proceeded to branch it out like that.

**Comment by tylerhyperHD** [comment]
I'm not saying you don't know how to code, i'm just saying it's a little unorganized and unprofessional.

**Comment by OxLemonxO** [comment]
meh sure i'll merge

**Comment by CreeperSeth** [comment]
A lot of code formatting is going to be fixed in 2.0. For example the code generator has already been properly formatted in 2.0.

File: src/main/java/me/unraveledmc/unraveledmcmod/ChatManager.java [hunk 2]
```diff
@@ -18,13 +18,14 @@
 
 public class ChatManager extends FreedomService
 {
+
     public static ChatColor acc = ChatColor.GOLD;
     public static boolean acr = false;
     public static boolean acn = false;
     public static final List<String> GRATIS_IPS = Arrays.asList(new String[]
-    	    {
```
Comment (CreeperSeth) [inline src/main/java/me/unraveledmc/unraveledmcmod/ChatManager.java]: This is just my code style. I prefer that to have indentation as it's not a method. Much like how when you indent string when they get too long, I have this indented as it signals to me that it's too long to be on one line.

**Pull request merged**